Pipfile

```diff
@@ -66,6 +66,7 @@ kombu = "==4.3"
 singledispatch = "==3.4.0.3"
 backports-abc = "==0.5"
 whitenoise = "*"
+django-redis = "*"
```

Comment:
Hmm should we pin this django-redis version, so it doesn't update everytime we run `pipfile.lock`?

Reply:
I'm hesitant about always pinning by default. I know pipenv can be a pain, but if we pin everything ourselves we are losing one of the big benefits of a system like pipenv, which is that we can indicate which packages it should be okay to update without having to update them all the time.

Reply:
If we keep running into those pipenv problems, then I may change my mind, though. ;-)